_data/authors.yml

```diff
@@ -47,6 +47,9 @@ nat_tarnoff:
 rachel_vasquez:
   name: Rachel R. Vasquez
   contact: http://rachievee.com
+ryan_r_smith:
```

Comment:
Please remove this. `authors.yml` is metadata reserved for people who have written a [post](https://github.com/a11yproject/a11yproject.com/tree/gh-pages/_posts).

Reply:
@ericwbailey Whoops... yep, I'll revert that change.